cli/src/domain/config.js

```diff
@@ -41,13 +43,12 @@ module.exports = (client, ui) => {
         }
     };
 
-    const get = function*(project, env, { configName = 'default', tag, hash }) {
```

Comment:
The `configName` is always "default".
It's an artifact from the project inception and will be removed slowly.